storybook/RechartsTheme.js

```diff
@@ -0,0 +1,38 @@
+import { create } from '@storybook/theming/create';
+import logo from './recharts_logo.png';
```

Comment:
The 'logo' isn't really a logo, its just an H1 with some styling 

See here https://github.com/recharts/recharts.org/blob/master/src/layouts/Frame.js#L63-L67

Not sure if that works in storybook's case though

Reply:
yeah I saw that - but Storybook requires an image. If you have one hosted somewhere I can link to that instead